### PR TITLE
Workshop 01-03. Code cleanup. TODOs updated.

### PR DESCRIPTION
**Pull request by webanimal**

### CODE REVIEW

File: app/src/main/java/com/android/fundamentals/workshop02/Workshop2Fragment.kt [hunk 1]
```diff
@@ -7,7 +7,7 @@ import androidx.fragment.app.Fragment
 import androidx.fragment.app.viewModels
 import com.android.fundamentals.R
 
-class Workshop2Fragment : Fragment(R.layout.fragment_workshop_1) {
+class Workshop2Fragment : Fragment(R.layout.fragment_workshop_2) {
```
Comment (rHlushan) [inline app/src/main/java/com/android/fundamentals/workshop02/Workshop2Fragment.kt]: workshop 1 and 2 use same layout
1 and 2 workshop implement same but in different ways
Maybe just rename fragment_workshop_1 -> fragment_workshop_1_workshop_2 and use it?

File: app/src/main/java/com/android/fundamentals/workshop02/Workshop2ViewModel.kt [hunk 2]
```diff
@@ -9,24 +9,27 @@ class Workshop2ViewModel(
     private val interactor: LoginInteractor
 ) : ViewModel() {
 
-    //TODO 06: Create private MutableLiveData for state storing. Set initial state State.Init
+    //TODO 06: Create private property MutableLiveData for state storing.
+    // Pass initial value "State.Init" in constructor: MutableLiveData(State.Init()).
```
Comment (rHlushan) [inline app/src/main/java/com/android/fundamentals/workshop02/Workshop2ViewModel.kt]: Init -> Default

File: app/src/main/java/com/android/fundamentals/workshop02/Workshop2ViewModel.kt [hunk 3]
```diff
@@ -9,24 +9,27 @@ class Workshop2ViewModel(
...
+            //TODO 08: Set "State.Loading()" to the private liveData's value.
 
             val loginResult = interactor.login(userName = userName, password = password)
 
-            //TODO 09: Handle loginResult and create new state depend on it (success or error)
+            //TODO 09: Handle "loginResult" with "when()".
+            // Set actual "State. ...()" to the private liveData's value (success, errors).
 
-            //TODO 10: Set new state to private liveData
+            //TODO 10: Set "State.Default()" to the private liveData's value.
```
Comment (rHlushan) [inline app/src/main/java/com/android/fundamentals/workshop02/Workshop2ViewModel.kt]: looks like
//TODO 10: Set "State.Default()" to the private liveData's value.
non needed

File: app/src/main/java/com/android/fundamentals/workshop01/Workshop1Presenter.kt [hunk 4]
```diff
@@ -28,14 +27,16 @@ class Workshop1Presenter(
...
 
             val loginResult = interactor.login(userName = userName, password = password)
-            //TODO 07: Handle loginResult and show success or error depend on it (using view methods)
-            /*when (loginResult) {
-            }*/
+            //TODO 07: Uncomment and Handle "loginResult".
+            // On "view" instance, show success or errors depending on result.
+            // To autocomplete, place cursor on "when" word, press "Alt+Enter", remaining branches.
+//            when (loginResult) {
+//            }
```
Comment (vadzim-vys) [inline app/src/main/java/com/android/fundamentals/workshop01/Workshop1Presenter.kt]: missing shift for `//`?

**Review by rHlushan — APPROVED** [review]

**Review by vadzim-vys — APPROVED** [review]

**Pull request merged**